test/drivers/test_suites/syscall_exit_suite/accept_x.cpp

```diff
@@ -210,7 +196,7 @@ TEST(SyscallExit, acceptX_UNIX)
 	evt_test->assert_numeric_param(4, (uint32_t)0);
 
 	/* Parameter 5: queuemax (type: PT_UINT32) */
-	evt_test->assert_numeric_param(5, (uint32_t)unix_max_queue_len);
+	evt_test->assert_numeric_param(5, (uint32_t)QUEUE_LENGTH);
```

Comment:
So looks like the "odd" behavior has gone now? At least on my system, changing to `QUEUE_LENGTH` now seems to work.

Reply:
ahahahaha unfortunately i left the comment only in the big PR, but there was a reason behind it https://github.com/falcosecurity/libs/pull/783#discussion_r1044598112